go-controller/pkg/ovn/gateway_init.go

```diff
@@ -50,13 +50,24 @@ func (oc *Controller) gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet,
 		Options:     logicalRouterOptions,
 		ExternalIDs: logicalRouterExternalIDs,
 	}
+
+	if oc.loadBalancerGroupSupport {
+		lbGroups, err := oc.findLoadBalancerGroup()
```

Comment:
why not just store the UUID in "oc" (the controller) when you create the load balancer group? Then you dont have to find it again in other places in the code.

Reply:
Makes sense, thanks for the suggestion!